### DISCUSSION TITLE
Goreleaser ignoring homebrew configuration

### DISCUSSION THREAD

**Discussion by david-mkl** [goreleaser/goreleaser#2814] [Q&A] [answered]
I can't seem to get goreleaser to publish to homebrew.

My `.goreleaser.yml` can be [found here](https://github.com/david-mk-lawrence/html-to-md/blob/main/.goreleaser.yaml). The homebrew tap repo is [here](https://github.com/david-mk-lawrence/homebrew-html-to-md).

When I run `goreleaser release --rm-dist`, the package is succesfully released to github without error, but nothing is committed to the homebrew repo, and there are no errors from goreleaser.

I've also run `goreleaser release --snapshot --rm-dist` and inspected the generated `dist` folder. The generated `config.yml` is missing the `brews` section, so it seems goreleaser is completely ignoring it. I feel like I'm just missing something obvious in the `.goreleaser.yml` file, but I've tried several different iterations without any luck. Any help would be appreciated.

**Comment by caarlos0** [comment, marked as answer]
your config is invalid

```
❯ goreleaser check
   • running goreleaser v1.4.1-pro
   • loading config file       file=.goreleaser.yml
   ⨯ command failed            error=failed to load config: yaml: line 9: did not find expected key
```

plrease refer to the [documentation](https://goreleaser.com), make sure `goreleaser check` passes, and see if the issue is still happening.
